### PR TITLE
Survey command and info, more documentation and bugfixes

### PR DESCRIPTION
**Pull request by JerBouma**
This PR does the following:
- Add a command `survey` that opens the most recent survey.
- Add a link within the documentation to fill in the most recent survey.
- Add a link in the exit message about filling in the survey.
- For the bugfix for `about`, I noticed that within the `ta` menu typing `about` doesn't work since it adds in "terminal" twice. This was a very minor fix from changing an if-statement to elif.
- Also update documentation for `settings`, `featflags` and `sources` which now has a proper section in the Getting Started page.

The link, https://openbb.co/survey, still needs to be defined.

image

image

image

image

### CODE REVIEW

**Comment by minhhoang1023** [comment]
Thanks for setting this up Jer! 🙌 

@piiq @andrewkenreich A quick question about the installer! My idea is that we can add the Survey link in the exit message for the Terminal. @JerBouma mentioned that it will only appear when people type `q` to exit. Is there any way we can still display the message when people exit in a different way?

**Comment by piiq** [comment]
> Is there any way we can still display the message when people exit in a different way?

If the user closes the window the window just closes and there is no exit per se. We can display survey link in the start message, or in some other place

**Comment by minhhoang1023** [comment]
It throws an error when I launched the command.

image

File: terminal.py [hunk 1]
```diff
@@ -246,6 +249,10 @@ def call_guess(self, other_args: List[str]) -> None:
             )
             console.print(f"[red]{e}[/red]")
 
+    def call_survey(self) -> None:
```
Comment (minhhoang1023) [inline terminal.py:252]: since we are not using any class attribute, it can be a static class here.

**Review by minhhoang1023 — CHANGES_REQUESTED** [review]

**Comment by JerBouma** [comment]
Fixed the survey command not working.  Does it matter much it isn't a static method?

**Comment by minhhoang1023** [comment]
@JerBouma it's software engineering best practice. So that an instance doesn't have to be instantiated each time you call it.

It would take less than 1 min to fix it, and we can keep the code base to a high-quality standard.

**Comment by minhhoang1023** [comment]
Awesome! Working now 🚀

**Comment by JerBouma** [comment]
> 

Could you suggest how?

**Comment by minhhoang1023** [comment]
For the survey, we could add "Take 2 mins to fill in our survey" or "Fill in our 2-minute survey". Letting people know that this is extremely QUICK to do!

Looks great overall

**Comment by JerBouma** [comment]
image

image

Better?

**Comment by minhhoang1023** [comment]
> > @JerBouma it's software engineering best practice. So that an instance doesn't have to be instantiated each time you call >it.
> > It would take less than 1 min to fix it, and we can keep the code base to a high-quality standard.
> 
> Could you suggest how?

Just need to add the @staticmethod and remove the self. Like this

```
    @staticmethod
    def call_survey(_) -> None:
        """Process survey command"""
        webbrowser.open("https://openbb.co/survey")
```

**Comment by JerBouma** [comment]
Done done

**Review by minhhoang1023 — APPROVED** [review]
LGG

**Review by piiq — APPROVED** [review]

**Pull request merged**